scripts/visualize/visualize-doy.R

```diff
@@ -38,10 +41,13 @@ visualize.doy <- function(viz = getContentInfo(viz.id = "doy-NM")){
   #################
   xml_name(doy_svg, ns = character()) <- "g"
   xml_attr(doy_svg, "class") <- "axis-labels"
-
-  vb <- xml_attr(doy_svg, "viewBox")
+  vb_doy <- xml_attr(doy_svg, "viewBox")
```

Comment:
Looks like the axis lines that are in these figures don't end up being visible (and they are also lumped into the `axis-labels`). Was the intent to leave them off?  If you want to style them, you may need to grab them and either group them with `<g class='axis-lines"/>` or something like that.